### PR TITLE
flask_iiif: python compatibility and fixes

### PR DESCRIPTION
**Pull request by drjova**
* FIX pep257 issues.

* FIX python 2.6 compatibility

Signed-off-by: Harris Tzovanakis <me@drjova.com>

### CODE REVIEW

**Comment by drjova** [comment]
WIP

**Comment by jirikuncar** [comment]
@drjova please amend the commit message:
```
global: python 2.9 compatibility and fixes

* Makes code compliant with PEP257.

* BETTER Improves python 2.6 compatibility.

* FIX Addresses issue with `cache` handler configuration.

Signed-off-by: Harris Tzovanakis <me@drjova.com>
```

**Comment by drjova** [comment]
:+1: thanks

File: flask_iiif/errors.py [hunk 1]
```diff
@@ -7,7 +7,7 @@
 # it under the terms of the Revised BSD License; see LICENSE file for
```
Comment (jirikuncar) [inline flask_iiif/errors.py]: please update copyright year

File: setup.py [hunk 2]
```diff
@@ -75,6 +75,7 @@ def run_tests(self):
         'redis',
         'six',
         'pillow',
+        'ordereddict',
```
Comment (jirikuncar) [inline setup.py]: please update copyright year

Comment (jirikuncar) [inline setup.py]: @drjova I don't see where you are using the `ordereddict` library.

Comment (jirikuncar) [inline setup.py]: @drjova why do we need `ordereddict` dependency?

Comment (drjova) [inline setup.py]: `ordereddict` is needed by `flask_restful` and is missing from python 2.6 that's the reason why it is failing [more details](https://travis-ci.org/inveniosoftware/flask-iiif/jobs/44354896#L726).

File: setup.py [hunk 3]
```diff
@@ -72,9 +72,9 @@ def run_tests(self):
         'Flask',
         'Flask-RESTful>=0.2.12',
         'blinker',
-        'redis',
         'six',
         'pillow',
+        'ordereddict',
```
Comment (jirikuncar) [inline setup.py]: if it's only for Python 2.6 then please do something similar to https://github.com/inveniosoftware/kwalitee/blob/master/setup.py#L33-L52. Thanks

**Pull request merged**